docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
```

Comment:
Remove this file... docker compose is for your local use and not everyone uses it.